morpho-checkpoint-node/src/Morpho/Tracing/Pretty.hs

```diff
@@ -132,8 +262,25 @@ instance
   where
   mpretty = viaShow
 
-instance (StandardHash blk) => MPretty (TraceChainSyncServerEvent blk) where
-  mpretty = viaShow
+instance MPretty (GenTx (MorphoBlock h c)) where
+  mpretty (MorphoGenTx tx _) = mpretty tx
```

Comment:
The second parameter should also be printed, in order for being able to easily correlate `TraceTxSubmissionOutbound` traces